DMTR-102.tex

```diff
@@ -1,40 +1,36 @@
 \documentclass[DM,lsstdraft,STR,toc]{lsstdoc}
 \usepackage{geometry}
 \usepackage{longtable,booktabs}
+\usepackage{enumitem}
+\usepackage{arydshln}
 
 \input meta.tex
 
+\providecommand{\tightlist}{
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
 \begin{document}
 
-\def\milestoneName{Raw Image Archiving}
-\def\milestoneId{LDM-503-8b}
-\def\product{LSST Raw Image Archiving System}
+\def\milestoneName{Raw image archiving for data from ccob device}
```

Comment:
CCOB, not ccob.

According to LDM-503, this is  “Small Scale CCOB Data Access”. Why is the milestone name changing here? If it needs to change, we should update P6 and the baselined documentation, too.

Reply:
done.  Updated LVV-P10

Reply:
fixed